src/ngraph/op/constant.hpp

```diff
@@ -39,6 +41,9 @@ namespace ngraph
                 static constexpr NodeTypeInfo type_info{"Constant", 0};
                 const NodeTypeInfo& get_type_info() const override { return type_info; }
                 Constant() = default;
+
+                Constant(const std::shared_ptr<runtime::HostTensor>& tensor);
```

Comment:
If you plan on sharing the tensor data you really need to document that else someone will change tensor